datasync/chaindatafetcher/kas/repository_contracts.go

```diff
@@ -33,7 +34,7 @@ func filterKIPContracts(api BlockchainAPI, event blockchain.ChainEvent) ([]*FT,
 	)
 	caller := newContractCaller(api)
 	for _, receipt := range event.Receipts {
-		if receipt.ContractAddress == (common.Address{}) {
+		if receipt.Status != types.ReceiptStatusSuccessful || receipt.ContractAddress == (common.Address{}) {
```

Comment:
Contracts will be filtered only when the receipt status is successful.

Reply:
I cannot get this line. I think it should be 
`if receipt.Status == types.ReceiptStatusSuccessful && receipt.ContractAddress == (common.Address{}) {`